tests/cephadm/test_verify_crash_reports.py

```diff
@@ -0,0 +1,48 @@
+from cli.cephadm.cephadm import CephAdm
+from cli.exceptions import OperationFailedError
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """Verify redeploy for a specific service
+    Args:
+        **kw: Key/value pairs of configuration information to be used in the test.
+    """
+    # Check for mgr nodes
+    installer = ceph_cluster.get_nodes(role="installer")[0]
+
+    # Identify the osd pid
+    cmd = "pgrep osd"
```

Comment:
can't we use - https://github.com/red-hat-storage/cephci/blob/master/cli/utilities/utils.py#L599 ? If no then create generic method and add ^^^